upload.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
-set -eu
+set -u
+
+for attempt in {1..5}; do
+    echo "Uploading #$attempt: $1"
+	case "${1##*.}" in
+		jsonld) ct=application/ld+json;;
+		ttl) ct=text/turtle;;
+		*) ct=text;; # fail it
+	esac
+    curl --silent -X POST -H "Content-Type: $ct" --data-binary "@$1"  ${GRAPHDB_API_URL:-http://localhost:8889/bigdata/sparql}
```

Comment:
@yarikoptic - I had to change to `http://localhost:7878/store?default` for oxigraph to see any data

Reply:
going to http://localhost:7878/store?default IIRC would immediately trigger download of something. web UI to send queries was on top URL IIRC.